R/tm_g_spiderplot.R

```diff
@@ -259,9 +283,9 @@ srv_g_spider <- function(id, data, filter_panel_api, reporter, dataname, label,
       vref_line <- input$vref_line
       href_line <- input$href_line
 
-      validate(need(iv$is_valid(), "Misspecification error: please observe red flags in the encodings."))
-      validate(need(nrow(ADSL) > 0, "ADSL data has zero rows"))
-      validate(need(nrow(ADTR) > 0, paste(dataname, "data has zero rows")))
+      # reference lines preprocessing
+      vref_line <- as_numeric_from_comma_sep_str(input$vref_line)
```

Comment:
Can remove

```
vref_line <- input$vref_line
href_line <- input$href_line
```

from just above here

Reply:
changed to `href_line <- as_numeric_from_comma_sep_str(href_line)` to maintain original flow